### ISSUE TITLE
testng-failed.xml does not carry over the parameters of methods from origin suite xml

### ISSUE THREAD

**Issue by ningzhangnj** [cbeust/testng#969]
Sample test case:

```
public class Test1 {
	@Test
	@Parameters({ "name", "description" })
	public void test1(@Optional("HisName") String name,
            @Optional("HisDescription") String description) {
		Assert.fail("Fail tc to generate testng-failed.xml");
	}
}
```

Sample origin suite xml:
```
<suite name="Test1 suite">	
	<test name="Test1 test">
		<classes>
			<class name="com.Test1">
			     <methods>
                    <include name="test1">                     
                        <parameter name="name" value="MyName" />
                        <parameter name="description" value="MyDescription" />   
                    </include>               
                </methods>
			</class>
		</classes>			
	</test>
</suite>
```
But the parameters of method are missed in testng-failed.xml:
```
<suite name="Failed suite [Test1 suite]">
  <test name="Test1 test(failed)">
    <classes>
      <class name="com.ericsson.Test1">
        <methods>
          <include name="test1" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.ericsson.Test1 -->
    </classes>
  </test> <!-- Test1 test(failed) -->
</suite> <!-- Failed suite [Test1 suite] -->
```

**Comment by testphreak** [comment]
Issue was fixed by @krmahadevan and should be in 6.9.11. https://github.com/cbeust/testng/issues/960

**Comment by ningzhangnj** [comment]
No, Fix for #960 resolve another issue.  Fix for #960 remove duplicated Suite in testng-failed.xml.  I want the parameters of the include method appear in testng-failed.xml.

**Comment by ningzhangnj** [comment]
Current generated testng-failed.xml is:
```
<suite name="Failed suite [Test1 suite]">
  <test name="Test1 test(failed)">
    <classes>
      <class name="com.ericsson.Test1">
        <methods>
          <include name="test1" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.ericsson.Test1 -->
    </classes>
  </test> <!-- Test1 test(failed) -->
</suite> <!-- Failed suite [Test1 suite] -->
```

Below testng-failed.xml is wanted:
```
<suite name="Failed suite [Test1 suite]">
  <test name="Test1 test(failed)">
    <classes>
      <class name="com.ericsson.Test1">
        <methods>
          <include name="test1" invocation-numbers="0">
               <parameter name="name" value="MyName" />
               <parameter name="description" value="MyDescription" />  
          </include>
        </methods>
      </class> <!-- com.ericsson.Test1 -->
    </classes>
  </test> <!-- Test1 test(failed) -->
</suite> <!-- Failed suite [Test1 suite] -->
```

**Comment by juherr** [comment]
Fixed by #975
